src/Core/Framework/Api/OAuth/AccessToken.php

```diff
@@ -81,19 +81,8 @@ public function initJwtConfiguration(): void
     {
         if ($this->privateKey instanceof FakeCryptKey) {
             $this->jwtConfiguration = $this->privateKey->configuration;
-
-            return;
         }
 
-        Feature::triggerDeprecationOrThrow('v6.7.0.0', 'The JWT generation should be done using HMAC');
-
-        // tag:v6.7.0.0 - throw an exception when the key is not our FakeCryptKey
-        /** @var non-empty-string $contents */
-        $contents = $this->privateKey->getKeyContents();
-        $this->jwtConfiguration = Configuration::forAsymmetricSigner(
-            new Sha256(),
-            InMemory::plainText($contents, $this->privateKey->getPassPhrase() ?? ''),
-            InMemory::plainText('empty', 'empty')
-        );
+        $this->jwtConfiguration = JWTConfigurationFactory::createJWTConfiguration();
```

Comment:
I think the else should throw an exception that the privateKey is wrong type of class

Reply:
no need to throw we can create a JWT config from scratch and set a fake private key 🤔 

Or am I missing something?